src/RadMarshak/test_radiation_marshak.cpp

```diff
@@ -62,17 +63,26 @@ template <> AMREX_GPU_HOST_DEVICE auto RadSystem<SuOlsonProblem>::ComputeRossela
 	return kappa;
 }
 
-template <> AMREX_GPU_HOST_DEVICE auto quokka::EOS<SuOlsonProblem>::ComputeTgasFromEint(const double /*rho*/, const double Egas) -> double
+static constexpr int nmscalars_ = Physics_Traits<SuOlsonProblem>::numMassScalars;
+amrex::ignore_unused(nmscalars_);
```

Comment:
You can't call functions outside of an enclosing scope. You can only declare variables. However, You should just get rid of  `nmscalars_` entirely, since it appears to be unused in this file, as far as I can tell?

Reply:
Oh, sorry, I see where it's used. I think it's a compiler bug if it complains about this being unused. Can you revert the code to the earlier version where it complains about this variable being unused?

Reply:
So, you mean go back to the version where I wasn't using `ignore_unsed()`?

Reply:
Yeah. It is probably a bug in Apple's compiler. It doesn't cause a warning with gcc. I will disable `WARNINGS_AS_ERRORS` for the macOS tests.